### DISCUSSION TITLE
[Feature Request] Show/Hide instance owner option for Discord Presence

### DISCUSSION THREAD

**Discussion by PigeonNotFound** [vrcx-team/VRCX#467] [Ideas]
As the title says, have an option to show or hide the instance owner's name.

**Ex:** Invite+ (_**VRCUser**_) (VR) (4 of 8)

Would be nice to have!

**Comment by Asovrix** [comment]
Would be cool, but I am not too sure if it's always possible to fetch that info, specifically for invite(+) and friends only instances made by someone you do not have on your friends list.

**Reply by PigeonNotFound** [reply]
Not sure how their API works either but if you're joining/within the instance, it should always shows the instance owner's user when you click on the instance info. and I don't think you can join a Friends-only instance unless you're friends with the instance owner.
